web/modules/custom/server_general/src/ElementWrapTrait.php

```diff
@@ -226,4 +226,25 @@ protected function wrapRoundedCornersBig(array $element): array {
     ];
   }
 
+  /**
+   * Remove nested empty arrays.
+   *
+   * If the element is an array of arrays, we'd like to remove empty ones.
+   * However, if the element is a one dimension array, we'll skip it.
+   *
+   * @param array $element
+   *   The render array.
+   *
+   * @return array
+   *   The filtered render array.
+   */
+  protected function filterEmptyElements(array $element): array {
+    if (count($element) <= 1) {
+      // No change.
+      return $element;
+    }
+
+    return array_filter($element);
```

Comment:
This still filters out 

```php
$element = [
  '#theme' => 'server_theme_x',
  '#some_count' => '0',
  '#is_something' => FALSE,
];
```

because count $element here would return 3 so it will pass it on to `array_filter`. Perhaps we should do something like this:

```suggestion
    if (count(\Drupal\Core\Render\Element::properties($element))) {
      // Element has top level properties beginning with #.
      // Do not filter.
      return $element;
    }

    return array_filter($element);
```

Reply:
Thanks. I'll write a small test to validate